### DISCUSSION TITLE
Verefication Help (Need a Human)

### DISCUSSION THREAD

**Discussion by knights-coding** [community/community#137940] [GitHub Education]
### Hi everyone,

Any chance I could get a human to help verify my teacher account?

I keep uploading multiple documents that don't seem to meet the requirements, and at this point I can't even upload anymore, it only allows the camera to take a picture, which isn't going to help with anything that I possibly could provide at this point.

Assistance would be great. I'm near giving up at this point.

**Comment by exkludera** [comment]
You can try contacting support by opening a [ticket](https://support.github.com/contact/education)
